pydal/_load.py

```diff
@@ -5,10 +5,12 @@
     from collections import OrderedDict
 except:
     from .contrib.ordereddict import OrderedDict
+    
 try:
+# try external module
+    import simplejson as json
```

Comment:
From simplejson page: 
" simplejson is the externally maintained development version of the json library included with Python 2.6 and Python 3.0, but maintains backwards compatibility with Python 2.5."

Btw, this is already patched in py3 branch.